buildutils/OSX_PostBuild.py

```diff
@@ -0,0 +1,79 @@
+# On OSX, ld builds libraries that contain hard coded path to their dependencies.
+# After installation on a client machiene, these paths become invalid unless we 
+# modify them to reference @rpath.  This script adds @rpath!
+
```

Comment:
I wonder what mechanism we used before to tackle this problem?

Reply:
Me too.